src/pages/components/variants/inputCheckbox/WithHelp.tsx

```diff
@@ -1,21 +1,34 @@
 import { useState } from 'react';
 import { InputCheckbox } from '@boston-scientific/anatomy-react';
 import Example from 'shared/components/Example';
+import { Fieldset } from '@boston-scientific/anatomy-react';
 
 const WithHelp = (): JSX.Element => {
   const [isChecked, setIsChecked] = useState(true);
 
   return (
-    <Example>
-      <div className="bsds-form-control">
-        <InputCheckbox
-          label="Checkbox"
+    <>
+      <Example>
+        <div className="bsds-form-control">
+          <InputCheckbox
+            label="Checkbox"
+            helpText="This is an example of help text. It can wrap to two lines, but try not to go longer than three."
+            checked={isChecked}
```

Comment:
should this be checked by default? we're not showing it as checked by default in SB or in other examples